src/Tests/TestDefaults.php

```diff
@@ -0,0 +1,9 @@
+<?php declare(strict_types=1);
+
+namespace Swag\Braintree\Tests;
+
+readonly final class TestDefaults
+{
+    public const CURRENCY_ID = 'b7d2554b0ce847cd82f3ac9bd1c0dfca';
+    public const SHOP_ID = 'shop-id';
```

Comment:
can't we omit them if the ahve the ids collection always containing the random generated ones?